### PR TITLE
Style: be succinct and positive, use contractions and sentence case

### PR DESCRIPTION
**Pull request by hugovk**
For #3167.

The Material Design styleguide says to [be succinct](https://material.io/design/usability/accessibility.html#writing):

> Keep content and accessibility text short and to the point. Screen reader users hear every UI element read aloud. The shorter the text, the faster the screen reader users can navigate it.

And [be positive](https://web.archive.org/web/20180410101124/https://material.io/guidelines/style/writing.html):

image

And use contractions:

image

And:

> Avoid using dashes to separate text.

And use sentence case:

image

### CODE REVIEW

File: tests/functional/legacy_api/test_removed.py [hunk 1]
```diff
@@ -17,8 +17,8 @@
 def test_removed_upload_apis(webtest, action):
     resp = webtest.post("/legacy/?:action={}".format(action), status=410)
     assert resp.status == \
-        ("410 Project pre-registration is no longer required or supported, so "
-         "continue directly to uploading files.")
+        ("410 Project pre-registration is no longer required or supported, "
+         "please upload your files.")
```
Comment (pradyunsg) [inline tests/functional/legacy_api/test_removed.py]: `upload your files directly.`

(no please)

File: tests/unit/forklift/test_legacy.py [hunk 2]
```diff
@@ -1081,8 +1080,8 @@ def test_fails_with_admin_flag_set(self, pyramid_config, db_request):
         resp = excinfo.value
 
         assert resp.status_code == 403
-        assert resp.status == ("403 New Project Registration Temporarily "
-                               "Disabled See "
+        assert resp.status == ("403 New project registration temporarily "
```
Comment (pradyunsg) [inline tests/unit/forklift/test_legacy.py]: `New Project Registration temporarily`

File: tests/unit/forklift/test_legacy.py [hunk 3]
```diff
@@ -1429,8 +1428,8 @@ def test_upload_fails_with_legacy_ext(self, pyramid_config, db_request):
 
         assert resp.status_code == 400
         assert resp.status == (
-            "400 Invalid file extension. PEP 527 requires one of: .egg, "
-            ".tar.gz, .whl, .zip (https://www.python.org/dev/peps/pep-0527/)."
+            "400 Invalid file extension. Use .egg, .tar.gz, .whl or .zip "
+            "extension (PEP 527 https://www.python.org/dev/peps/pep-0527)."
```
Comment (pradyunsg) [inline tests/unit/forklift/test_legacy.py]: `400 Invalid file extension: Use .egg, .tar.gz, .whl or .zip extension. (PEP 527 https://www.python.org/dev/peps/pep-0527)`

or

`400 Invalid file extension: Use .egg, .tar.gz, .whl or .zip extension as per PEP 527 (https://www.python.org/dev/peps/pep-0527).`

File: tests/unit/forklift/test_legacy.py [hunk 4]
```diff
@@ -3084,8 +3083,8 @@ def test_submit(pyramid_request):
 
     assert resp.status_code == 410
     assert resp.status == \
-        ("410 Project pre-registration is no longer required or supported, so "
-         "continue directly to uploading files.")
+        ("410 Project pre-registration is no longer required or supported, "
+         "please upload your files.")
```
Comment (pradyunsg) [inline tests/unit/forklift/test_legacy.py]: Noted above.

File: tests/unit/forklift/test_legacy.py [hunk 5]
```diff
@@ -2057,8 +2056,8 @@ def test_upload_fails_with_invalid_extension(self, pyramid_config,
 
         assert resp.status_code == 400
         assert resp.status == (
-            "400 Invalid file extension. PEP 527 requires one of: .egg, "
-            ".tar.gz, .whl, .zip (https://www.python.org/dev/peps/pep-0527/)."
+            "400 Invalid file extension. Use .egg, .tar.gz, .whl or .zip "
```
Comment (pradyunsg) [inline tests/unit/forklift/test_legacy.py]: Noted above.

File: warehouse/forklift/legacy.py [hunk 6]
```diff
@@ -465,7 +466,7 @@ class MetadataForm(forms.Form):
                     "bdist_dmg", "bdist_dumb", "bdist_egg", "bdist_msi",
                     "bdist_rpm", "bdist_wheel", "bdist_wininst", "sdist",
                 ],
-                message="Unknown type of file.",
+                message="Use known file type.",
```
Comment (pradyunsg) [inline warehouse/forklift/legacy.py]: `Use a known file type`

File: warehouse/forklift/legacy.py [hunk 7]
```diff
@@ -733,7 +734,7 @@ def file_upload(request):
     if request.flags.enabled('read-only'):
         raise _exc_with_message(
             HTTPForbidden,
-            'Read Only Mode: Uploads are temporarily disabled',
+            'Read-only mode: uploads are temporarily disabled',
```
Comment (pradyunsg) [inline warehouse/forklift/legacy.py]: `Uploads`

File: warehouse/forklift/legacy.py [hunk 8]
```diff
@@ -846,7 +847,7 @@ def file_upload(request):
         if request.flags.enabled('disallow-new-project-registration'):
             raise _exc_with_message(
                 HTTPForbidden,
-                ("New Project Registration Temporarily Disabled "
+                ("New project registration temporarily disabled. "
```
Comment (pradyunsg) [inline warehouse/forklift/legacy.py]: Noted above.

File: warehouse/forklift/legacy.py [hunk 9]
```diff
@@ -1050,8 +1051,8 @@ def file_upload(request):
     if _dist_file_regexes[project.allow_legacy_files].search(filename) is None:
         raise _exc_with_message(
             HTTPBadRequest,
-            "Invalid file extension. PEP 527 requires one of: .egg, .tar.gz, "
-            ".whl, .zip (https://www.python.org/dev/peps/pep-0527/)."
+            "Invalid file extension. Use .egg, .tar.gz, .whl or .zip "
```
Comment (pradyunsg) [inline warehouse/forklift/legacy.py]: Noted above.

File: warehouse/templates/base.html [hunk 10]
```diff
@@ -210,9 +210,9 @@ <h3>Get help</h3>
             <h3>About PyPI</h3>
           </li>
           {% if request.registry.settings.get("statuspage.url") %}
-          <li><a href="https://status.python.org/">Status: <span data-statuspage-domain="{{ request.registry.settings['statuspage.url'] }}">All Systems Operational</span></a></li>
+          <li><a href="https://status.python.org/">Status: <span data-statuspage-domain="{{ request.registry.settings['statuspage.url'] }}">all systems operational</span></a></li>
```
Comment (pradyunsg) [inline warehouse/templates/base.html]: No changes needed here?

File: warehouse/templates/base.html [hunk 11]
```diff
@@ -210,9 +210,9 @@ <h3>Get help</h3>
...
           {% endif %}
-          <li><a href="https://www.python.org/dev/peps/pep-0541/">Package Index Name Retention</a></li>
+          <li><a href="https://www.python.org/dev/peps/pep-0541/">Package index name retention</a></li>
```
Comment (pradyunsg) [inline warehouse/templates/base.html]: No changes needed here?

File: warehouse/templates/base.html (hunk 10 above)
Comment (hugovk) [reply warehouse/templates/base.html]: [Material Design](https://web.archive.org/web/20180410101124/https://material.io/guidelines/style/writing.html#writing-tone) says:

> **Use sentence-style caps**
>
> Titles, headings, labels, and menu items should use sentence-style capitalization (capitalizing only the first word of a sentence).
> 
> Avoid capitalizing all letters except where the Material spec requires them, such as Button text.

And I don't think it's a proper noun, just the current status. But happy to change it if needed, let me know!

File: warehouse/templates/base.html (hunk 11 above)
Comment (hugovk) [reply warehouse/templates/base.html]: I'm less certain about this. One could argue it's a proper noun, the name of the PEP (or is the name "PEP 541"?). Or is it just the title of the PEP and they follow a different capitalisation style over there?

https://www.python.org/dev/peps/pep-0541/

File: warehouse/templates/base.html (hunk 10 above)
Comment (hugovk) [reply warehouse/templates/base.html]: Here is it in context:

image

https://pypi.org/

File: tests/unit/forklift/test_legacy.py [hunk 12]
```diff
@@ -1081,8 +1080,8 @@ def test_fails_with_admin_flag_set(self, pyramid_config, db_request):
         resp = excinfo.value
 
         assert resp.status_code == 403
-        assert resp.status == ("403 New Project Registration Temporarily "
-                               "Disabled See "
+        assert resp.status == ("403 New Project Registration temporarily "
```
Comment (di) [inline tests/unit/forklift/test_legacy.py]: I think we should sentence-case this entire line.

File: tests/unit/forklift/test_legacy.py [hunk 13]
```diff
@@ -1655,7 +1654,7 @@ def test_upload_fails_with_invalid_digest(self, pyramid_config, db_request,
 
         assert resp.status_code == 400
         assert resp.status == (
-            "400 The digest supplied does not match a digest calculated "
+            "400 The digest doesn't match a digest calculated "
```
Comment (di) [inline tests/unit/forklift/test_legacy.py]: I think we are losing some context here about what the error is.

File: tests/unit/forklift/test_legacy.py [hunk 14]
```diff
@@ -2057,8 +2056,8 @@ def test_upload_fails_with_invalid_extension(self, pyramid_config,
 
         assert resp.status_code == 400
         assert resp.status == (
-            "400 Invalid file extension. PEP 527 requires one of: .egg, "
-            ".tar.gz, .whl, .zip (https://www.python.org/dev/peps/pep-0527/)."
+            "400 Invalid file extension: Use .egg, .tar.gz, .whl or .zip "
+            "extension. (PEP 527 https://www.python.org/dev/peps/pep-0527)"
```
Comment (di) [inline tests/unit/forklift/test_legacy.py]: Probably can just lose the "PEP 527" here entirely

File: tests/unit/forklift/test_legacy.py [hunk 15]
```diff
@@ -3084,8 +3083,8 @@ def test_submit(pyramid_request):
 
     assert resp.status_code == 410
     assert resp.status == \
-        ("410 Project pre-registration is no longer required or supported, so "
-         "continue directly to uploading files.")
+        ("410 Project pre-registration is no longer required or supported, "
+         "upload your files.")
```
Comment (di) [inline tests/unit/forklift/test_legacy.py]: Maybe "upload your files instead"?

File: warehouse/forklift/legacy.py [hunk 16]
```diff
@@ -279,7 +279,8 @@ def _validate_project_url_list(form, field):
 
 
 def _validate_rfc822_email_field(form, field):
-    email_validator = wtforms.validators.Email(message='Invalid email address')
+    email_validator = wtforms.validators.Email(
+        message='Use valid email address')
```
Comment (di) [inline warehouse/forklift/legacy.py]: "Use a valid email address"?

File: warehouse/forklift/legacy.py [hunk 17]
```diff
@@ -298,13 +299,13 @@ def _raise(message):
 
     charset = parameters.get('charset')
     if charset and charset != 'UTF-8':
-        _raise("charset is not valid")
+        _raise("Use valid charset")
```
Comment (di) [inline warehouse/forklift/legacy.py]: "Use a valid charset"?

File: warehouse/forklift/legacy.py [hunk 18]
```diff
@@ -335,7 +336,7 @@ class MetadataForm(forms.Form):
                 #       claims it is producing a Metadata 2.0 metadata when in
                 #       reality it's more like 1.2 with some extensions.
                 ["1.0", "1.1", "1.2", "2.0", "2.1"],
-                message="Unknown Metadata Version",
+                message="Use known metadata version.",
```
Comment (di) [inline warehouse/forklift/legacy.py]: "Use a known metadata version"?

File: warehouse/forklift/legacy.py [hunk 19]
```diff
@@ -298,13 +299,13 @@ def _raise(message):
...
     if charset and charset != 'UTF-8':
-        _raise("charset is not valid")
+        _raise("Use valid charset")
 
     variant = parameters.get('variant')
     if (content_type == 'text/markdown' and variant and
             variant not in _valid_markdown_variants):
         _raise(
-            "variant is not valid, expected one of {}".format(
+            "Use valid variant, expected one of {}".format(
```
Comment (di) [inline warehouse/forklift/legacy.py]: "Use a valid variant"?

File: warehouse/forklift/legacy.py [hunk 20]
```diff
@@ -375,7 +376,7 @@ class MetadataForm(forms.Form):
             wtforms.validators.Length(max=512),
             wtforms.validators.Regexp(
                 r"^.+$",  # Rely on the fact that . doesn't match a newline.
-                message="Multiple lines are not allowed.",
+                message="Use single line only.",
```
Comment (di) [inline warehouse/forklift/legacy.py]: "Use a single line only"?

File: warehouse/templates/base.html (hunk 11 above)
Comment (di) [reply warehouse/templates/base.html]: I think this change is fine.

File: warehouse/templates/base.html (hunk 10 above)
Comment (di) [reply warehouse/templates/base.html]: I think this change is fine.

File: tests/unit/forklift/test_legacy.py (hunk 12 above)
Comment (hugovk) [reply tests/unit/forklift/test_legacy.py]: Done.

File: tests/unit/forklift/test_legacy.py (hunk 13 above)
Comment (hugovk) [reply tests/unit/forklift/test_legacy.py]: Reverted.

File: tests/unit/forklift/test_legacy.py (hunk 14 above)
Comment (hugovk) [reply tests/unit/forklift/test_legacy.py]: Done.

File: tests/unit/forklift/test_legacy.py (hunk 15 above)
Comment (hugovk) [reply tests/unit/forklift/test_legacy.py]: Done.

File: warehouse/forklift/legacy.py (hunk 16 above)
Comment (hugovk) [reply warehouse/forklift/legacy.py]: Done.

File: warehouse/forklift/legacy.py (hunk 17 above)
Comment (hugovk) [reply warehouse/forklift/legacy.py]: Done.

File: warehouse/forklift/legacy.py (hunk 18 above)
Comment (hugovk) [reply warehouse/forklift/legacy.py]: Done.

File: warehouse/forklift/legacy.py (hunk 19 above)
Comment (hugovk) [reply warehouse/forklift/legacy.py]: Done.

File: warehouse/forklift/legacy.py (hunk 20 above)
Comment (hugovk) [reply warehouse/forklift/legacy.py]: Done

**Pull request merged**